src/pages/version/VersionTabs.tsx

```diff
@@ -109,9 +109,9 @@ export const VersionTabs: React.FC<Props> = ({
       taskCount,
       childPatches,
       numFailedChildPatches,
-      patchId: id,
+      patchId: versionId,
```

Comment:
changing this parameter to be called `versionId` might make this a bit less confusing to read